modules/nf-core/software/gatk4/gatherbqsrreports.nf

```diff
@@ -3,7 +3,7 @@ include { initOptions; saveFiles; getSoftwareName } from './../functions'
 params.options = [:]
 def options    = initOptions(params.options)
 
-process GATK_GATHERBQSRREPORTS {
+process GATK4_GATHERBQSRREPORTS {
     label 'memory_singleCPU_2_task'
```

Comment:
would label'process_low' work here too to re-use as many nf-core labels as possible?

Reply:
Good point, my plan was to remove all sarek labels and switch to the one commonly use in modules